com.unity.multiplayer.mlapi/Tests/Runtime/Metrics/Ownership/NetworkMetricsOwnershipChangeDispatchTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections;
+using System.Linq;
+using MLAPI.Metrics;
+using MLAPI.RuntimeTests.Metrics.NetworkVariables;
+using NUnit.Framework;
+using Unity.Multiplayer.NetworkProfiler;
+using Unity.Multiplayer.NetworkProfiler.Models;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace MLAPI.RuntimeTests.Metrics.Ownership
+{
+    public class NetworkMetricsOwnershipChangeDispatchTests
+    {
+        NetworkManager m_NetworkManager;
+        NetworkMetrics m_NetworkMetrics;
+
+        Guid m_GameObjectId;
+
+        [SetUp]
+        public void SetUp()
+        {
+            NetworkManagerHelper.StartNetworkManager(out m_NetworkManager);
+            m_NetworkMetrics = m_NetworkManager.NetworkMetrics as NetworkMetrics;
+
+            m_GameObjectId = NetworkManagerHelper.AddGameNetworkObject("ObjectToChangeOwner");
+            NetworkManagerHelper.SpawnNetworkObject(m_GameObjectId);
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            NetworkManagerHelper.ShutdownNetworkManager();
+        }
+        
+        [UnityTest]
+        public IEnumerator TrackOwnershipChangeSentMetric()
+        {
+            var waitForMetricValues = new WaitForMetricValues<OwnershipChangeEvent>(m_NetworkMetrics.Dispatcher, MetricNames.OwnershipChangeSent);
+
+            NetworkManagerHelper.InstantiatedNetworkObjects[m_GameObjectId].ChangeOwnership(1);
```

Comment:
Should we call this with `m_NetworkManager.LocalClientId` instead to make sure it lines up with the asserts at the end?